src/validation.cpp

```diff
@@ -1851,6 +1851,18 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_NULLDUMMY;
     }
 
+    // SEGWIT2X signalling.
+    if ( VersionBitsState(pindex->pprev, chainparams.GetConsensus(), Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE &&
```

Comment:
All right.  Can you/someone clarify the other issue?  Will the current code start signaling bit 1 support as soon as bit 4 is locked in (rather than 2w later when it activates), and if not, shouldn't it?